app/views/admin/includes/_admin_toolbar.html.erb

```diff
@@ -64,8 +64,28 @@
     <%= link_to t( 'list' ), users_path %>
   <% end %>
     |
+<% end %>
 
+<% if @discussion.present? %>
+  <%= t( 'admin.discussion.title' ) %>:
+<% if current_user_can? :hide, :discussions %>
+  <% if @discussion.hidden? %>
+    <%= link_to t( 'unhide' ), unhide_discussion_path( @discussion ) %>
```

Comment:
Try to avoid generating `HTTP GET` links/anchor tags that make state changes. They're susceptible to link prefetching wreaking havoc (admittedly unlikely). Prefer using `button_to` which generates a button wrapped with a form. Also prefer to use an HTTP method that conveys its mutating a resource such as PATCH.

This applies to all the hide/unhide and lock/unlock links for discussions and comments in this PR.